libmaxminddb/plan.sh

```diff
@@ -0,0 +1,72 @@
+pkg_origin=core
+pkg_name=libmaxminddb
+pkg_version=1.3.2
+pkg_license=('Apache')
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source="https://github.com/maxmind/libmaxminddb/releases/download/${pkg_version}/${pkg_name}-${pkg_version}.tar.gz"
+pkg_shasum="e6f881aa6bd8cfa154a44d965450620df1f714c6dc9dd9971ad98f6e04f6c0f0"
+pkg_upstream_url="https://github.com/maxmind/libmaxminddb"
+pkg_description="C library for the MaxMind DB file format"
+pkg_build_deps=(
+  core/coreutils
+  core/gcc
+  core/make
+  core/perl
+  )
+pkg_deps=(core/glibc)
```

Comment:
Adding `core/gcc-libs` to `pkg_deps` will fix the pthreads error when running with DO_CHECK.